### DISCUSSION TITLE
Unable to refer event bridge custom rule deployed via CF

### DISCUSSION THREAD

**Discussion by anudeepjusanu** [serverless/serverless#11551] [Q&A]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest v3 release?

- [ ] Yes, I'm using the latest v3 release

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

HI,
I am trying to refer an existing event bus rule which is setup by the cloud formation, I am unable to refer the rule, can anyone please help me out on this. This is what I am trying to do

events:
- eventBridge:
eventBus: eventBusARN
name: ruleName
pattern:
account:
- ${aws:accountId}

### Service configuration (serverless.yml) content

```yaml
N/A
```

### Command name and used flags

N/A

### Command output

```shell
N/A
```

### Environment information

```shell
N/A
```

**Comment by medikoo** [comment]
Moving to discusionss
